### ISSUE TITLE
unable to start new game

### ISSUE THREAD

**Issue by redgoblin88** [tales/tales-client#49]
![Uploading Screenshot_2016-08-02-20-02-32.png…]()

I have just got the game today, I opened it up, made a new charecter as prompted but it claimed it already existed, I exited and saw thatupon making an account you are automatically given a charecter.

from here I decided to join and actually play but was given an error as follows:
"No game server found for the map the charecter is on" which is indicated in the screenshot given

**Comment by bjorn** [comment]
> I have just got the game today, I opened it up, made a new charecter as prompted but it claimed it already existed, I exited and saw thatupon making an account you are automatically given a charecter.

Hmm, actually no character is automatically created, so something must have gone wrong there.

> from here I decided to join and actually play but was given an error as follows:
> "No game server found for the map the charecter is on" which is indicated in the screenshot given

I'm getting the same error currently. There must be a configuration error on the server.

Note that at the moment there is no developer actually having time for working on this game. @Ablu only barely manages to at least keep the server running. There is still a lot to do on the game before it is nicely playable.

**Comment by Ablu** [comment]
The server was having some weird issues. It looked like some weird DNS 
issues. Also the game server crashed.

I restarted the whole server and run the game server inside gdb now.

Should be all working now again.

Regards,
Erik

**Comment by redgoblin88** [comment]
It is indeed all working now and explains why my charecter was naked, unfortunately I can see that the game may need a little bit more development as there is some latencies in spawning/ddspawning NPCs and the gear spawns sepperately plus changing equipment takes some time, but at first glance it looks like an amazing game and it is already complete, just those minor delays to reduce

**Comment by Ablu** [comment]
Yes. This delays are our main engine issue currently.

Especially with fighting the delays become unacceptable.

I have a series of patches in
https://github.com/Ablu/manaserv/commits/architecture_playground in
order to try to make the server more modular. The idea is to be able to
share the logic between server and client. Then the client could predict
stuff like fighting. However there still is a lot to do. I certainly
still plan to work on this, but currently I am busy with finishing my
bachelor thesis. But after that I hope to have time for it again...

I wonder, how did you figure out about this game? We did not really do
any advertising at this point. Especially the android alpha is well
hidden at the moment :P (or did you build it on the pc?)

**Comment by redgoblin88** [comment]
I was looking on google play not too long ago as I was wanting a dwarven themed fantasy mmorpg (as im into dwarves) but the list of mmorpg games was short and the "source of tales" game happened to appear there but was shown as "early alpha" or something, I thought I should give it a try and to help you if possible with any issues if I find it good, which first impressions are really nice graphics despite that small server hickup.

**Comment by Ablu** [comment]
Oh... So the game showed up for you without you manually joining the
alpha channel first? So far we deliberately did not put the game into
the public release channel so people to do not get disappointed of it.
But you said you have been able to install it?

Anyway, thanks a lot for the input. It is always interesting to know how
people find out about the game.

Regards,
Ablu

**Comment by Ablu** [comment]
Alright. We realized that Google apparently made it easier to join the alpha channel directly via the playstore leading in a lot of installs.

Since we are aware that the game currently gives a bad first impression we disabled the open alpha now. You now need to be part of this [G+ Community](https://plus.google.com/communities/105749851870064248799) in order to register as an alpha tester.

Anyway I guess this issue is resolved.
